echopype/echodata/zarr_combine.py

```diff
@@ -936,7 +937,7 @@ def combine(
 
         Parameters
         ----------
-        zarr_path: str
+        zarr_path: Path
```

Comment:
I think this would push us backward to no cloud capability if it only accept `Path`.